pallets/vector/src/lib.rs

```diff
@@ -798,5 +888,13 @@ pub mod pallet {
 		fn is_domain_valid(domain: u32) -> bool {
 			WhitelistedDomains::<T>::get().contains(&domain)
 		}
+
+		fn get_function_ids() -> Result<(H256, H256), DispatchError> {
+			if let Some(function_ids) = FunctionIds::<T>::get() {
```

Comment:
Unimportant subjective style comment: Let-else would be a better choice here.

Reply:
yeah i get that, i sill feel this is simple and readable tbh